### ISSUE TITLE
Add search via voice option in app

### ISSUE THREAD

**Issue by ujjwalagrawal17** [commons-app/apps-android-commons#1640]
**Summary:**
Voice Searching can be added for searching categories/images.

Link: https://developer.android.com/guide/topics/search/search-dialog#VoiceSearch

**Comment by domdomegg** [comment]
Done in #1716

**Comment by ujjwalagrawal17** [comment]
Umm, This option isn't implemented in #1716

**Comment by domdomegg** [comment]
Yep you're correct, my bad. I think I initially thought it had because of the `voiceSearchMode` attribute (see https://github.com/commons-app/apps-android-commons/pull/1716/files#diff-aa7791505fe0795bce32c5176a32c707R42), but it doesn't seem to actually work yet.
